tests/testthat/helper-scope.R

```diff
@@ -251,7 +251,7 @@ renv_tests_init_packages <- function() {
   renv_namespace_load("remotes")
 
   # pak needs a little special handling
-  if (config$pak.enabled() && renv_package_installed("pak")) {
```

Comment:
I had to revert this change in order for the tests to pass locally for me.

Reply:
These tests, or something else? https://github.com/rstudio/renv/commit/bdafa4033ec3166aa1f36a4f02db8e3d09ca4ae0